app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -928,6 +940,7 @@ public String toJSON() {
             jsonObject.put("raw_calculated", raw_calculated);
             jsonObject.put("raw_data", raw_data);
             jsonObject.put("calculated_value_slope", calculated_value_slope);
+            jsonObject.put("calibration_uuid", calibration_uuid);
```

Comment:
To answer my own question, yes we do appear to know ahead of time when a json entry is going to be part of a batch so we could exclude the calibration uuid in that case? 

Seems like the least impact all round solution with only a slight increase in complexity, eg adding a boolean to indicate reduced size desired of the json record and add a wrapper for the default signature when a boolean is not present for the original behavior?